### DISCUSSION TITLE
Add redundancy to API methods

### DISCUSSION THREAD

**Discussion by sigier** [losol/eventuras#579] [General]
Option 1
Add  "includedetails=true" into request

Other options - TBC

**Comment by losolio** [comment]
I think for now we will just use this method! And when the new front-end is built, we could work with a better v4-api!? Perhaps graphql is a more suitable way to request data for frontend framework?

**Reply by sigier** [reply]
Right, and it worth to mention, that it is available in backend flavor too. Demo link: https://learn.microsoft.com/en-us/shows/on-net/getting-started-with-hotchocolate

**Comment by sigier** [comment]
AutoMapper should also be considered for cleaner code and, in some cases, for data shaping and performance improvement.
